builder/assets/clever.service

```diff
@@ -1,12 +1,11 @@
 [Unit]
 Description=Clever ROS package
-Requires=roscore.service
-After=roscore.service
+After=network.target
 
 [Service]
 User=pi
-EnvironmentFile=/lib/systemd/system/roscore.env
-ExecStart=/opt/ros/kinetic/bin/roslaunch clever clever.launch --wait --screen
+ExecStart=/bin/bash -c ". /home/parallels/catkin_ws/devel/setup.bash; \
+                        ROS_HOSTNAME=`hostname`.local exec roslaunch clever clever.launch --wait --screen --skip-log-check"
```

Comment:
I'm on the fence about this. At the very least we should document how this suffix works (and when we could omit it altogether).

Reply:
Are you talking about `--skip-log-check`?

Reply:
No, I was talking about `ROS_HOSTNAME` `.local` suffix (even though we did have the suffix for quite some time now, it's less obvious here). Some network configurations will work without specifying the Avahi hostname, this should probably be mentioned.